src/commands/remoteDebug/getRemoteDebugLanguage.ts

```diff
@@ -26,13 +30,22 @@ export function checkForRemoteDebugSupport(siteConfig: SiteConfigResource, conte
         if (version.startsWith('node')) {
             const splitVersion = version.split('|');
             if (splitVersion.length > 1 && isNodeVersionSupported(splitVersion[1])) {
-                // Node version is supported, so return successfully
-                return;
+                return RemoteDebugLanguage.Node;
+            } else {
+                throw new Error('Azure Remote Debugging is currently only supported for Node.js version >= 8.11 on Linux.');
```

Comment:
Not blocking, but this error and the error on line 47 should ideally be the same message.